src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -312,8 +312,15 @@ bool ov::pass::ConvertPrecision::run_on_model(const std::shared_ptr<ngraph::Func
         return false;
 
     bool has_fp16_compression = m_precisions.count(element::f32) > 0 && m_precisions[element::f32] == element::f16;
+    bool has_quantize_nodes = false;
+    for (const auto& op : f->get_ops()) {
+        if (std::dynamic_pointer_cast<opset8::FakeQuantize>(op)) {
```

Comment:
```suggestion
        if (as_typ_ptr<opset8::FakeQuantize>(op)) {
```

Reply:
these lines have been removed, but applied change dynamic_pointer_cast -> as_type_ptr in other modified places